### PR TITLE
IZE-105 ize help is reviewed updated extended (env, deploy, init)

### PR DESCRIPTION
**Pull request by psihachina**
## Changelog:
- expand descriptions of global flags
- add long descriptions and examples to env command
- add descriptions and examples to deploy commands

### Not required

### CODE REVIEW

File: internal/commands/deploy/deploy.go [hunk 1]
```diff
@@ -40,16 +40,16 @@ type Infra struct {
 
 var deployLongDesc = templates.LongDesc(`
 	Deploy infraftructure or sevice.
-	For deploy service the service name must be specimfied. 
+	For deploy service the service name must be specified.
```
Comment (kobrikx) [inline internal/commands/deploy/deploy.go:43]: Service name must be specified for a service deploy.

File: internal/commands/deploy/deploy_infra.go [hunk 2]
```diff
@@ -24,6 +25,22 @@ type DeployInfraOptions struct {
 	Terraform terraformInfraConfig
 }
 
+var deployInfraLongDesc = templates.LongDesc(`
+	Deploy only infraftructure.
```
Comment (kobrikx) [inline internal/commands/deploy/deploy_infra.go:29]: Only deploy infrastructure

File: internal/commands/ize.go [hunk 3]
```diff
@@ -85,9 +85,9 @@ func newApp() (*cobra.Command, error) {
 
 	rootCmd.PersistentFlags().StringP("log-level", "l", "", "enable debug messages")
 	rootCmd.PersistentFlags().StringP("config-file", "c", "", "set config file name")
-	rootCmd.PersistentFlags().StringP("env", "e", "", "set enviroment name")
-	rootCmd.PersistentFlags().StringP("aws-profile", "p", "", "set AWS profile")
-	rootCmd.PersistentFlags().StringP("aws-region", "r", "", "set AWS region")
+	rootCmd.PersistentFlags().StringP("env", "e", "", "(required) set enviroment name (overrides value set in ENV / IZE_ENV if any of them are set)")
```
Comment (kobrikx) [inline internal/commands/ize.go:88]: environment

**Review by kobrikx — CHANGES_REQUESTED** [review]
Look good, but need to change some typos.

**Review by igorkotof — APPROVED** [review]

**Review by AutomationD — APPROVED** [review]
LGTM

**Pull request merged**